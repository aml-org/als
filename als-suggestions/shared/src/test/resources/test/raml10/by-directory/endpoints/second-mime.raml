#%RAML 1.0
title: MyAPI
version: 1.0

/e:
  post:
    body:
      application/xml:
        type: string
      a*