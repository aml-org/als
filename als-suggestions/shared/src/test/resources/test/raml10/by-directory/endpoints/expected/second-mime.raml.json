[
  {
    "label": "application/json",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "application/json",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11application/json",
    "filterText": "application/json:\n  ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 9,
          "character": 6
        },
        "end": {
          "line": 9,
          "character": 7
        }
      },
      "newText": "application/json:\n  "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]